gulpfile.js

```diff
@@ -3,94 +3,110 @@ var gulp = require('gulp')
   , uglify = require('gulp-uglify')
   , sourcemaps = require('gulp-sourcemaps')
   , rename = require('gulp-rename')
-  , clean = require('gulp-clean')
+  , del = require('del')
   , usemin = require('gulp-usemin')
-  , cssmin = require('gulp-cssmin')
-  , livereload = require('gulp-livereload')
   , ghPages = require('gulp-gh-pages')
-  , runSequence = require('run-sequence')
   , babel = require('gulp-babel')
   , eslint = require('gulp-eslint')
+  , browsersync = require('browser-sync').create()
+  , postcss = require('gulp-postcss')
+  , cssnano = require('cssnano')
 
-// Watch Our Files
-gulp.task('watch', function() {
-    livereload.listen();
-    gulp.watch('src/**/*', ['build']).on('change', livereload.changed)
-    //gulp.watch('dist/**/*').on('change', livereload.changed)
-});
 
+// WATCH TASKS START
+function watchFiles() {
+    gulp.watch(["src/**/*"], {queue: false}, gulp.series(build, browserSyncReload))
```

Comment:
If you add a file extension to this watch process, we should be able to have it only watch `.es6.js` and `.css` files, preventing the infinite app.js loop. I _think_ that would look like:

```
["src/**/*.es6.js", "src/**/*.css"]
```